docs/installation/manually-setup.md

```diff
@@ -75,14 +75,22 @@ Then edit the config/setup/yurt-controller-manager.yaml
 
 ### 3.2 Setup Yurt-App-Manager
 
-please get `config/setup/all_in_one.yaml` from [yurt-app-manager repo](https://github.com/openyurtio/yurt-app-manager/tree/master/config/setup)
+Add OpenYurt Helm Reop：
+
+```shell
+helm repo add openyurt https://openyurtio.github.io/openyurt-helm
 ```
-$ kubectl apply -f config/setup/all_in_one.yaml
+
+Install `yurt-app-manager` use command as follows:
+
+```shell
+helm upgrade --crate-namespace openyurt --install yurt-app-manager openyurt/yurt-app-manager
 ```
 
-Wait for the yurt-app-manager operator to be created successfully
+Check `yurt-app-manager` pod if startup:
+
 ```
-$ kubectl get pod -n kube-system | grep yurt-app-manager
+kubectl get pod -n openyurt | grep yurt-app-manager
```

Comment:
maybe it's better to install `yurt-app-manager` in `kube-system` namespace

Reply:
@rambohe-ch OpenYurt's component should not share namespace with kube-system. We should treat as a standalone component.

Reply:
@huiwq1990 I think we should deploy OpenYurt components in kube-system namespace on default mode, and create a new namespace named openyurt. meanwhile, end user can deploy OpenYurt components in other namespace if they want.